update-permissions.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
note: this script is in dry-run by default, given the main purpose is to enable submitters to do local check. Hence I do not want them to have to specify a switch for the nominal use case (and it's a no-brainer to do this specifically in the `Jenkinsfile`)